### ISSUE TITLE
Incorrect ARM binary packaged in x86_64 deb

### ISSUE THREAD

**Issue by n0madic** [aerospike/php-client#38]
Hello,

I have encountered a significant issue with the aerospike-php-client-1.0.2-x86_64.deb package. It appears that the package, which is intended for x86_64 systems, mistakenly contains a binary built for the ARM architecture. This error prevents the library from loading properly on x86_64 systems.

```bash
# uname -a
Linux 2d8d78c51834 6.5.0-15-generic #15-Ubuntu SMP PREEMPT_DYNAMIC Tue Jan  9 22:39:36 UTC 2024 x86_64 GNU/Linux
```

Error encountered after install:
```
PHP Warning:  PHP Startup: Unable to load dynamic library 'libaerospike_php.so' (tried: /usr/lib/php/20220829/libaerospike_php.so (/usr/lib/php/20220829/libaerospike_php.so: cannot open shared object file: No such file or directory), /usr/lib/php/20220829/libaerospike_php.so.so (/usr/lib/php/20220829/libaerospike_php.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

Verification:
```bash
# file /usr/lib/php/20220829/libaerospike_php.so
/usr/lib/php/20220829/libaerospike_php.so: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[sha1]=a5ac0b4ccf183361086bd8014fa6ffef81a901d4, not stripped
```

**Comment by vmsachin** [comment]
Hi @n0madic, thanks for brining this to our attention. We'll take a look at it 👍

**Comment by vmsachin** [comment]
fixed in v1.1.0
